### DISCUSSION TITLE
App that accepts multiple connections

### DISCUSSION THREAD

**Discussion by mariosant** [dokku/dokku#5047] [Q&A] [answered]
I got a utility app that accepts both web and mqtt connections on different ports. Can this application run on Dokku?

**Comment by josegonzalez** [comment, marked as answer]
Yes but you won't be able to load balance to it with the built-in nginx proxy plugin as we only support http and https proxying. You'd need to find another plugin (or build one) to handle proxying on your behalf.

**Reply by mariosant** [reply]
Thanks for responding @josegonzalez
